csv-conduit.cabal

```diff
@@ -96,7 +96,7 @@ library
     , mtl
     , mmorph
     , primitive
-    , resourcet              >= 1.1.2.1
+    , resourcet              >= 1.1.2.1 && < 1.2
```

Comment:
I also noticed that csv-conduit does not build with resourcet-1.2, so I added this upper bound.